docarray/array/array_stacked.py

```diff
@@ -97,16 +94,13 @@ def _from_columns(
     def _create_columns(
         cls: Type[T], docs: DocumentArray, tensor_type: Type['AbstractTensor']
     ) -> Dict[str, Union['TorchTensor', T, NdArray]]:
+        from docarray.util._typing import is_tensor_union
```

Comment:
Why the lazy import?

Reply:
ah I need to fix. I had circular import so I did this